src/RolloutScorer/Readme.md

```diff
@@ -0,0 +1,80 @@
+# Rollout Scorer
+
+The Rollout Scorer is a tool for generating rollout scorecards.
+
+## Tool Description
+The rollout scorer takes two commands: `score` and `upload`.
+
+### `score`
+
+The `score` command calculates a particular rollout's score and generates the scorecard which it outputs as a CSV for review. Optionally, it can skip this output step and upload the results directly.
```

Comment:
Does "rolloutscorer help score" return this text? I prefer tools be self documenting for the most part (more likely to be up to date, don't have versioning problems, all that good stuff), and then it doesn't need to be here.

Reply:
I moved it to there and will delete these sections.